### DISCUSSION TITLE
How to close applications as an action?

### DISCUSSION THREAD

**Discussion by H-Tevin** [Codectory/AutoActions#46] [Q&A] [answered]
There is a program on my computer which interferes with games when it runs, however, I want it running all the time that certain games aren't running. Is there a way for AutoHDR to use the Run action to close the application upon launching a certain game and then run the application again when the game closes?

I've only been able to figure out how to run an application as an action, not close one. Maybe it's something in the arguments?

**Comment by Codectory** [comment, marked as answer]
Right now, it’s not possible to close other processes directly, but you can
run other scripts or processes with can do that for you.

Tevin Hendess ***@***.***> schrieb am Fr. 3. Dez. 2021 um
03:58:
